From 7efdda596055f3143cf5b4f32f23ec4b3dacc270 Mon Sep 17 00:00:00 2001
From: Tom Hochstein <tom.hochstein@nxp.com>
Date: Fri, 24 Aug 2018 17:40:29 -0500
Subject: [PATCH 10/22] uapi: Add ion.h to userspace

Signed-off-by: Tom Hochstein <tom.hochstein@nxp.com>
[Quanyang: The imx gstreamer plugins need use ion as memory allocation,
so we need to export ion.h to userspace. Or else imx gstreamer apps which
use vpudec plugin can't work and will throw out the error as below:
0:00:03.316022413   631 0xaaaabfe2ac50 WARN vpuallocator gstvpuallocator.c:125:gst_vpu_copy_phys_mem: TODO: use relevant hardware to accelerate memory copying!
And this patch is picked from:
https://source.codeaurora.org/external/imx/imx-manifest -b imx-linux-sumo -m imx-4.14.78-1.0.0_ga.xml
meta-fsl-bsp-release/imx/meta-bsp/recipes-kernel/linux/]
Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 include/uapi/linux/ion.h | 1 +
 1 file changed, 1 insertion(+)
 create mode 120000 include/uapi/linux/ion.h

diff --git a/include/uapi/linux/ion.h b/include/uapi/linux/ion.h
new file mode 120000
index 000000000000..17e8dbb24265
--- /dev/null
+++ b/include/uapi/linux/ion.h
@@ -0,0 +1 @@
+../../../drivers/staging/android/uapi/ion.h
\ No newline at end of file
-- 
2.17.1

